site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,37 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* IP Access Control for API access or API keys
+** This allows domains or API addresses to be safelisted to use an API key.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/933[GitHub Issue #933]
+* FusionAuth wasn't awesome enough, so we added a robust Threat Detection feature for enterprise customers. This feature includes: 
```

Comment:
The IP Access Control described above also falls within this bucket of features licensed by "Threat detection".  So if you want to call out the "feature of Threat Detection" - maybe you lead with that, and then list out features under it? (as I think you have done with Captcha, etc)